behandlingslager/domene/src/main/java/no/nav/ung/sak/behandlingslager/tilkjentytelse/InntektAbonnement.java

```diff
@@ -0,0 +1,100 @@
+package no.nav.ung.sak.behandlingslager.tilkjentytelse;
+
+import jakarta.persistence.AttributeOverride;
+import jakarta.persistence.AttributeOverrides;
+import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Version;
+import no.nav.k9.felles.konfigurasjon.konfig.Tid;
+import no.nav.ung.sak.behandlingslager.BaseEntitet;
+import no.nav.ung.sak.behandlingslager.PostgreSQLRangeType;
+import no.nav.ung.sak.behandlingslager.Range;
+import no.nav.ung.sak.domene.typer.tid.DatoIntervallEntitet;
+import no.nav.ung.sak.typer.AktørId;
+import org.hibernate.annotations.Type;
+
+import java.time.LocalDate;
+
+@Entity(name = "InntektAbonnement")
+@Table(name = "INNTEKT_ABONNEMENT")
+public class InntektAbonnement extends BaseEntitet {
```

Comment:
Denne må registreres i en orm-xml fil for at Hibernate skal klare å plukke den opp. Se filer i prosjektet som slutter med orm.xml, jeg antar vi vil ha en egen orm-xml fil for denne